scripts/policy/frameworks/cluster/nodes-experimental/manager.zeek

```diff
@@ -0,0 +1,55 @@
+##! This script provides experimental features in the cluster manager.
+
+@load base/frameworks/cluster
+@load policy/frameworks/cluster/experimental
+
+module Cluster::Experimental;
+
+global fully_connected_nodes_pending: set[string];
+
+event zeek_init()
+	{
+	fully_connected_nodes_pending = table_keys(Cluster::nodes);
+	}
+
+event node_fully_connected(name: string, id: string, resent: bool)
+	{
+	# If a node has already seen the cluster connected, the manager most likely
```

Comment:
Just a suggestion to help make the leap from "if a node has seen" to `resent` being `T`:
```suggestion
	# If a node resends this event, it has already seen the cluster connected. The manager most likely
```